modules/capi-yaml-machine-set-aws.adoc

```diff
@@ -16,28 +16,35 @@ kind: MachineSet
 metadata:
   name: <machine_set_name> # <1>
   namespace: openshift-cluster-api
+  labels:
+    cluster.x-k8s.io/cluster-name: <cluster_name> # <2>
 spec:
   clusterName: <cluster_name> # <2>
   replicas: 1
   selector:
     matchLabels:
       test: example
+      cluster.x-k8s.io/cluster-name: <cluster_name>
+      cluster.x-k8s.io/set-name: <machine_set_name>
   template:
     metadata:
       labels:
         test: example
+        cluster.x-k8s.io/cluster-name: <cluster_name>
+        cluster.x-k8s.io/set-name: <machine_set_name>
```

Comment:
same above, the label `cluster.x-k8s.io/set-name: <machine_set_name>` is machine label instead of machineset, here should be `machine.openshift.io/cluster-api-cluster: <cluster_name>`

Reply:
Same as above, this is a template for the labels that will be on the machine that gets created by this machineset. CAPI labels are correct here, see: https://issues.redhat.com/browse/OCPCLOUD-2880